lib/common-types.ts

```diff
@@ -148,7 +148,7 @@ export interface Adapter {
     params: GetSpaceParams
     userAgent: string
   }): Promise<any>
-  // makeRequest<R = unknown>(options: MakeRequestOptions): Promise<R>
+  makeRequest(options: MakeRequestOptions): Promise<any>
```

Comment:
q: what was the problem with unknown here?

Reply:
We get type issues in the plain client as `unknown` is not assignable to another type.
e.g. this fails:
```typescript
const varUnknown: unknown = 5;
const varString: string = varUnknown;
```